src/isIsogram.test.js

```diff
@@ -6,4 +6,29 @@ describe('isIsogram', () => {
   it(`should be declared`, () => {
     expect(isIsogram).toBeInstanceOf(Function);
   });
+
+  it(`should return true for a word with all unique letters`, () => {
+    expect(isIsogram('playgrounds'))
+      .toBe(true);
+  });
+
+  it(`should return false for a word with repeating letters`, () => {
+    expect(isIsogram('look'))
+      .toBe(false);
+  });
+
+  it(`should return false for a word with case-insensitive duplicates`, () => {
+    expect(isIsogram('Adam'))
+      .toBe(false);
+  });
+
+  it(`should return true for an empty string`, () => {
+    expect(isIsogram(''))
+      .toBe(true);
+  });
+
+  it(`should return false for a word with repeating letters`, () => {
+    expect(isIsogram('Oops'))
+      .toBe(false);
+  });
```

Comment:
The description for this test case is not unique. It is identical to the description on line 15. Consider changing it to something like `should return false for a word with repeating letters and mixed case` to make it distinct and clear.